test/rest_request_template/test_string_json_ignore_tombstone.json

```diff
@@ -10,8 +10,7 @@
     "snowflake.database.name":"SNOWFLAKE_DATABASE",
     "snowflake.schema.name":"SNOWFLAKE_SCHEMA",
     "key.converter":"org.apache.kafka.connect.storage.StringConverter",
-    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
```

Comment:
the corresponding default tombstone e2e ingestion test uses SnowflakeJsonConverter

Reply:
what will be the behavior if it is org.apache.kafka.connect.json.JsonConverter? Do we have a test for that?

Reply:
JsonConverter treats empty string as a tombstone record whereas SnowflakeJsonConverter treats it as a normal record.

We don't have snowpipe e2e tests with JsonConverter, but we do have IT tests that [test all converters](https://github.com/snowflakedb/snowflake-kafka-connector/blob/master/src/test/java/com/snowflake/kafka/connector/internal/TombstoneRecordIngestionIT.java#L140) with snowpipe

Reply:
Please double confirm that we dont have any behavior changes in snowpipe. Thanks